inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -26,12 +26,13 @@ LayerTestsCommon::LayerTestsCommon() : threshold(1e-2f), abs_threshold(-1.f) {
 }
 void LayerTestsCommon::ResizeNgraphFunction() {
     auto params = function->get_parameters();
+    std::map<std::string, ngraph::PartialShape> shapes;
     ASSERT_LE(params.size(), targetStaticShapes[index].size());
     for (size_t i = 0; i < params.size(); i++) {
-        params[i]->set_partial_shape(targetStaticShapes[index][i]);
+        shapes.insert({*params[i]->get_output_tensor(0).get_names().begin(), targetStaticShapes[index][i]});
```

Comment:
@ilyachur It's quite bulky construction. Could we improve it? 
CC @ilya-lavrenov

Reply:
@ilyachur can we have a helper `get_name()` which returns a first name in the set?

Reply:
Yes I will add the helper.
But I prefer to have `get_output_tensor_names()` instead of `get_name()`.

Reply:
Or maybe:
```c++
function->output().get_name();
```
The idea is to have any single name instead of set of names